### PR TITLE
Added support matrices

### PR DESCRIPTION
**Pull request by amathews-amd**
for docker releases

### CODE REVIEW

**Review by saadrahim — CHANGES_REQUESTED** [review]
Looks good but we should discuss the long term strategy for this page.

File: docs/release/support_matrix.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# Frameworks Support Matrix
+
+The software support matrices for ROCm container releases is listed. 
+
+## ROCm 5.6
+
+### Pytorch
+
+#### Ubuntu+ rocm5.6_internal_testing+169530b
+
```
[the remark this answers is not in the document]
Comment (amathews-amd) [reply docs/release/support_matrix.md:11]: The containers would only be released 1-2 weeks after ROCm 5.6

File: docs/release/support_matrix.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# Frameworks Support Matrix
+
+The software support matrices for ROCm container releases is listed. 
+
+## ROCm 5.6
```
[the remark this answers is not in the document]
Comment (amathews-amd) [reply docs/release/support_matrix.md:6]: We need to keep the info on file for all versions of ROCm that is publicly available.

File: docs/release/support_matrix.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+### Pytorch
+
```
[the remark this answers is not in the document]
Comment (amathews-amd) [reply docs/release/support_matrix.md:9]: cc: @amd-aakash

**Comment by SamuelMarks** [comment]
Did you also want to document kernel support like on https://docs.amd.com/bundle/ROCm-Release-Notes-v5.4.3/page/About_This_Document.html ?

BTW: I wanted to mess around with 6.2.8 kernel support for my  AMD Radeon™ RX 6850M XT 12GB GDDR6; i.e., Navi22 by hacking around [unofficial] patches referenced from https://github.com/RadeonOpenCompute/ROCm/issues/1668 at https://github.com/xuhuisheng/rocm-build.

**Comment by SamuelMarks** [comment]
Did you also want to document kernel support like on https://docs.amd.com/bundle/ROCm-Release-Notes-v5.4.3/page/About_This_Document.html ?

BTW: I wanted to mess around with 6.2.8 kernel support for my  AMD Radeon™ RX 6850M XT 12GB GDDR6; i.e., Navi22 by hacking around [unofficial] patches referenced from https://github.com/RadeonOpenCompute/ROCm/issues/1668 at https://github.com/xuhuisheng/rocm-build. - Specifically I want to know what kernel(s) are supported by this new version of ROCm and [offtopic] if you had any tips for porting to the latest kernel.

**Comment by xuhuisheng** [comment]
@SamuelMarks 
My suggesetion is using upstream kernel of amdgpu version.
Its said that the latest version of kernel had already hold an merged amdgpu-dkms version. If kernel version too new to be supported by ROCm offically, we could use this built-in amdgpu-dkms version directly.

**Comment by SamuelMarks** [comment]
Thanks @xuhuisheng I'll give that a shot

File: docs/release/support_matrix.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (saadrahim) [reply docs/release/support_matrix.md:11]: Need to document process for this step on post release actions.

**Comment by saadrahim** [comment]
> Did you also want to document kernel support like on https://docs.amd.com/bundle/ROCm-Release-Notes-v5.4.3/page/About_This_Document.html ?
> 
> BTW: I wanted to mess around with 6.2.8 kernel support for my AMD Radeon™ RX 6850M XT 12GB GDDR6; i.e., Navi22 by hacking around [unofficial] patches referenced from #1668 at https://github.com/xuhuisheng/rocm-build. - Specifically I want to know what kernel(s) are supported by this new version of ROCm and [offtopic] if you had any tips for porting to the latest kernel.

@SamuelMarks I won't block this PR to address your concerns. However, #1995 was raised to track this.

File: docs/release/support_matrix.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (saadrahim) [reply docs/release/support_matrix.md:9]: Tracked in #1996

**Review by saadrahim — APPROVED** [review]
Concerns tracked separately

**Pull request merged**